app/src/main/java/com/example/stromtracker/ui/geraete/auswertung/GeraeteAuswertungFragment.kt

```diff
@@ -129,12 +124,12 @@ class GeraeteAuswertungFragment(
             pie = initPieChart(pie)
 
             pie.data(data)
-            gesamtverbrauch =
+            gesamtverbrauchkWh =
                 roundTo2Decimal(verbraucherList.sumByDouble { geraete -> geraete.getJahresverbrauch() })
```

Comment:
Ich implementiere gerade, dass das nur gerundet in die Datenbank kommt, also sobald alles gemerged ist kann das dann weg.

Reply:
Wird beim nächsten commit entfernt.